corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/util.js

```diff
@@ -151,9 +159,7 @@ hqDefine("cloudcare/js/formplayer/utils/util", function () {
             }
             // Selections only deal with strings, because formplayer will send them back as strings
             if (_.isArray(selection)) {
-                var selectedValues = (sessionStorage.selectedValues !== undefined) ? JSON.parse(sessionStorage.selectedValues) : {};
-                selectedValues[sessionStorage.queryKey] = selection.join(',');
-                sessionStorage.selectedValues = JSON.stringify(selectedValues);
+                hqImport("cloudcare/js/formplayer/utils/util").setSelectedValues(selection);
```

Comment:
I just noticed this now. Is it necessary to use `hqImport` here? Isn't the `setSelectedValues` function defined in the same JS file as this?

Reply:
@snopoke ah great question I tested it out and it seems unnecessary if I instead either say `this.setSelectedValues` or `Utils.setSelectedValues`. @orangejenny do you have any insights on what best practice is? I see down below on [line 188](https://github.com/dimagi/commcare-hq/pull/32094/files#diff-0fccf35567df7b182bf4be6561b133e25373f1f88e0da42f7d4fecf603223f41L188)  it follows the import convention

Reply:
Either will work, but it's odd to import a module into itself, and it might break when this is switched to use RequireJS. I'd use `Util` here for clarity. `this` gets confusing in our js because we use so many callbacks that its definition changes often, so our best practice is to assign it to a variable and use that.

Reply:
Line 188 should also use `Util`. I blamed a bit but didn't figure out where that was introduced, wondering if it was moved from another file.